
std_msgs/Header header
int64 seq
int64 sensor_seq
std_msgs/Float64MultiArray probabilities
int32 id
string approach
